### PR TITLE
Added a section about Presentations

### PR DESCRIPTION
**Pull request by j-holub**
Sorry for the second Pull Request. I accidentally committed my changes to master and did a pull request on the master branch. Sorry about that.

I have studied computer science for one semester in Japan. We always had 発表会s to present our research progress. The term was used very often in that field.

Then I tried to find some more useful words in that context. Most of them are from a blog post about a Keynote Template a Japanese friend of me created. That's why I confirm that they are really used by natives.

I'd say not only in research but also within software companies presentations are important and it'd be nice to have some vocabulary on that matter.

### CODE REVIEW

**Comment by ronkorving** [comment]
Good stuff, thanks! I wonder if these words cannot be put under existing categories though. Some are technical (Color scheme, Font, Template <- at least not limited to presentations) while others may be compatible with "Business"? What do you think?

**Comment by j-holub** [comment]
I agree with you that some of these words are not limited to presentations. The presentation stuff could go very well under business, though there are a lot of computer science research conferences all over the world, with presentations about non-business related stuff. But I still think business would be fine.

Terms like Color scheme, Font and Template can be considered technical, but they are also somewhat design terms (which is far away from technical I think).

It's very difficult to fit these terms into categories. You could eben put them under a category like "Front End Development" for example. If I had to put them in an existing category I'd probably go with Software Development with Front End in my mind.

**Comment by ronkorving** [comment]
So, thinking about "when we allow a new category", I think my own answer would be "When there are at least 3 words in said category, and an obvious potential for more in the future". And of course, the content has to fit the overarching theme of being developer-oriented.

Given that, perhaps presentations/conferences/events, etc. *do* deserve their own category. I can already imagine some words we can easily add to it. So let's go for it.

So how about under "Presentations":

- To present
- Presentation
- Presentation (Meeting) (you made a typo in this PR btw: "presenation")
- Slide

And under the existing "Software development":

- Color Scheme
- Font
- Template (I wonder if in Japanese it's as widely applicable as in English..?)

**Comment by j-holub** [comment]
That sounds really good actually.

As for template I don't know either. I got it from a blog post of a Japanese friend of mine about his Keynote Theme he created himself. He's a programmer, too so I'm just gonna ask him if it can be used for something like a *C++ Template* as well.

(Sorry 'bout the typo btw)

**Comment by ronkorving** [comment]
Great. Then if you could update this PR, I'll happily merge it 👍

**Comment by j-holub** [comment]
According to this article https://cpprefjp.github.io/lang/cpp11/variadic_templates.html I'd assume the テンプレート also means C++ Template.

I updated the pull request. Still not sure about **Color Scheme** though. Could be more relevant to Presentation but I think it's fine either way.

**Comment by ronkorving** [comment]
Agreed. This works 👍

**Comment by j-holub** [comment]
Just wondering if you were planning on merging it :)

File: ReadMe.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ guidelines though, to keep things consistent and avoid it exploding into a copy
 
 - [Computer And Application Usage](#computer-and-application-usage)
 - [Devices](#devices)
+- [Presenations](#presentations)
```
Comment (ronkorving) [inline ReadMe.md]: You've still labeled this "Presenations" (typo)

**Comment by ronkorving** [comment]
I will, thanks for reminding :) Been out of it for a bit, with the holidays and all :)

**Comment by j-holub** [comment]
Oh sorry, I thought I had changed the typo.

Happy new year by the way :)

**Pull request merged**